### ISSUE TITLE
Linter warns of bad selector style erroneously

### ISSUE THREAD

**Issue by CJ-Wright** [conda-forge/conda-smithy#596]
I am getting the message
`Selectors are suggested to take a <two spaces>#<one space>[<expression>] form.` but all my selectors are correct, see [here](https://github.com/conda-forge/pyfai-feedstock/pull/16/files#diff-e178b687b10a71a3348107ae3154e44cL16)

**Comment by jakirkham** [comment]
I think it is referring to this [line]( https://github.com/conda-forge/pyfai-feedstock/pull/16/files#diff-e178b687b10a71a3348107ae3154e44cR29 ), which looks kind of like a selector. The message is to try and ensure selectors appear in comments (i.e. preceded with `#`). Not sure what we can actually do to fix that. 🤔

That all said, I don't think I have seen that style of entrypoint. Would that even work with `conda-build`?

**Comment by CJ-Wright** [comment]
I pulled that from the [setup.py](https://github.com/silx-kit/pyFAI/blob/master/setup.py#L774)

**Comment by jakirkham** [comment]
Right, I just don't know if `conda-build` knows how to handle that. Does that entrypoint work ok?
